common/src/test/java/eu/seaclouds/common/apps/AbstractSeaCloudsAppTest.java

```diff
@@ -14,7 +14,7 @@
  *    See the License for the specific language governing permissions and
```

Comment:
I am not sure about if this test and the subclasses make sense in this PR, probably a new PR should be open.